### PR TITLE
Support for pr13

### PR DESCRIPTION
**Pull request by Tim-NL**

### CODE REVIEW

File: upload/modules/CraftingStore/module.php [hunk 1]
```diff
@@ -84,14 +84,17 @@ public function onPageLoad($user, $pages, $cache, $smarty, $navs, $widgets, $tem
...
+            PermissionEnum::DEFAULT => $this->language->get('moderator', 'staff_cp') . ' &raquo; ' . $this->craftingStoreLanguage->get(LanguageEnum::PREFIX, LanguageEnum::CRAFTING_STORE),
+            PermissionEnum::SETTINGS => $this->language->get('moderator', 'staff_cp') . ' &raquo; ' . $this->craftingStoreLanguage->get(LanguageEnum::PREFIX, LanguageEnum::SETTINGS),
         ]);
 
         // Navigation links
         $this->userNavigationBuilder->build($navs[NavigationEnum::USER_NAVIGATION]);
         $this->backendNavigationBuilder->build($navs[NavigationEnum::ADMIN_NAVIGATION], $user);
     }
+	
+	public function getDebugInfo(): array {
```
Comment (PadowYT2) [inline upload/modules/CraftingStore/module.php:96]: This could probably result an error as no `return[];`

**Review by PadowYT2 — COMMENTED** [review]
Also $queries weren't changed

**Comment by PadowYT2** [comment]
> > Also $queries weren't changed
> 
> Thanks! Could you share what queries might be an issue? I tried to reinstall the module after removing the tables and it created them as expected.
> 
> -Tim

Dang, I thought it would break. Anyways it was deprecated, see [here](https://github.com/NamelessMC/Nameless/blob/v2.0.0-pr13/core/classes/Database/Queries.php)

**Comment by Tim-NL** [comment]
Thank you, in that case, I'll leave it as-is for now and update it in a future release. I do not have a proper development environment for this project (this module) and as such, I do not have any type of completion or even syntax highlighting - That is why that obvious syntax error came into this PR. I'll set that up as well. 

-Tim

**Comment by PadowYT2** [comment]
> Thank you, in that case, I'll leave it as-is for now and update it in a future release. I do not have a proper development environment for this project (this module) and as such, I do not have any type of completion or even syntax highlighting - That is why that obvious syntax error came into this PR. I'll set that up as well.
> 
> -Tim

I can make a pr if you want to

**Comment by Tim-NL** [comment]
Feel free to create one if you want, as it is only deprecated at the moment it is not breaking and does not need immediate fixing. I'm okay with moving it at a later date, but if you want feel free to create a PR.

-Tim

**Pull request merged**